src/AppInstallerSharedLib/YamlWrapper.h

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
This file is added instead of moved?

Reply:
Looks like I forgot to delete it in CommonCore...